docs/operator/deploy.md

```diff
@@ -8,35 +8,38 @@ sidebarDepth: 2
 
 This document explains the workflow to have a working group of drand nodes generate randomness. There are four sections to this guide:
 
-1. Generate the long-term key pairs and the group file.
 1. Start the daemons.
-1. Have each node collectively participate in the distributed key generation (DKG).
-1. Generate randomness.
+2. Generate the long-term key pairs and the group file.
+3. Have each node collectively participate in the distributed key generation (DKG).
+4. Generate randomness.
+
+You can repeat these steps every time you want to start a new network for randomness generation. If the drand daemon is already running, please
+skip that step. For each new network, a unique identifier, known as `Beacon ID` is required. 
 
 ## Setup
 
 The setup process for a drand node consists of the following steps:
 
-1. Generate the long-term key pair for each node.
 1. Start the drand daemon on each node.
-1. The leader starts the command as a coordinator & every participant connects to the coordinator to setup the network.
+2. Generate the long-term key pair for each new network.
+3. The leader starts the command as a coordinator & every participant connects to the coordinator to setup the network.
 
 This document explains how to do the setup with the drand binary itself. If you want to install drand using Docker, follow the [Docker instructions instead](/operator/docker/).
 
 ### Long-term key
 
-Each drand node needs a public and secret key to interact with the rest of the network. To generate these keys run [`drand generate-keypair`](/operator/drand-cli/#drand-generate-keypair) followed by the address of your node:
+Each drand network needs a public and secret key to interact with the rest of the nodes. To generate these keys run [`drand generate-keypair`](/operator/drand-cli/#drand-generate-keypair) followed by the address of your node:
 
 ```bash
-drand generate-keypair drand.example.com
+drand generate-keypair --id {beacon-id} drand.example.com
```

Comment:
I would suggest to say how to choose the id. Specifically that the operator can choose anything as long as it hasn't been  chosen before.

Reply:
Done!